controller/array_action/ds8k_rest_client.py

```diff
@@ -74,6 +74,16 @@ def __init__(self, service_address, user, password,
 
         self._client = Client(**client_kwargs)
 
+    def is_valid(self):
+        try:
+            # Send command to check if client is still valid. IO enclosures is chosen since it is fast.
+            self._client.get_io_enclosures()
+            return True
+        except Exception as ex:
```

Comment:
shouldn't there be a specific error to expect here?
see: https://github.com/IBM/ibm-block-csi-driver/pull/135#discussion_r437391502

Reply:
Possible but since I don't know what is returned for example when token is up-to-date I prefer to handle any exception this way.
In  the attacked comment it can't happen since we create new connection there.

Reply:
why can't we know what is returned when token is (not) up-to-date?
besides the generally bad practice of a catch-all `Exception`, I think we risk all ds8k functionality if there would be any issue specifically with `get_io_enclosures` that is not related to authentication/connectivity.
and why can't it happen when we create a new connection there?

Reply:
1) get_io_enclosures  is list command without arguments which should never fall when connection is valid and storage is up
2) we can't check since i don't know when is its expiration. it may be days.,
3) This is bad practice but i don't know exactly which exceptions can happen here

Reply:
4. Token can't be expired a second after creating connection.

Reply:
1. I think `which should never fail` is not a good assumption 
2. just because you don't know, doesn't mean it cannot be checked
3. I think it's worth checking
4. so is it called there only right after creation? I thought it is used for later checks as well